packages/elements/src/calendar/__test__/calendar.defaults.test.js

```diff
@@ -2,11 +2,27 @@
 import '@refinitiv-ui/elements/calendar';
 
 import '@refinitiv-ui/elemental-theme/light/ef-calendar.js';
-import { elementUpdated, expect, fixture, nextFrame } from '@refinitiv-ui/test-helpers';
+import {
+  elementUpdated,
+  expect,
+  fixture,
+  fixtureSync,
+  nextFrame,
+  oneEvent
+} from '@refinitiv-ui/test-helpers';
 import { parse } from '@refinitiv-ui/utils';
 
-import { RenderView } from '../../../lib/calendar/constants.js';
-import { setMonthView, setYearView } from './utils.js';
+import { CalendarRenderView } from '../../../lib/calendar/constants.js';
```

Comment:
I think we missed this one, it's much better to import from package entrypoint.

```suggestion
import { CalendarRenderView } from '@refinitiv-ui/elements/calendar';
```

Reply:
good catch! Updated for both v6 and v7 PRs.